ci-build/docker-build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+shopt -s extglob
+
+TMP_DIR=$(mktemp -d)
+
+function main {
+  local here
+  here=$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )
+
+  cp "$here/Dockerfile" "$TMP_DIR"
+  cd "$here/.."
+  cp -r !(.*|html|output|Dockerfile) "$TMP_DIR"
+  cp .*.pl "$TMP_DIR"
+  cd "$TMP_DIR"
+  trap cleanTemp EXIT
+
+  local docker_hub_repo="whatwg/html-build"
+
+  # Build the Docker image, using Docker Hub as a cache. (This will be fast if nothing has changed
+  # in html-build or its dependencies).
+  docker pull whatwg/wattsi
+  docker pull ptspts/pdfsizeopt
+  docker pull "$docker_hub_repo" || true
```

Comment:
Why the `|| true` for this one, but not the previous? I guess whatwg/html-build didn't exist when you wrote this, but now it does. If you've intentionally left it to make bootstrapping into another place possible, then that's fine.

Reply:
Hmm, yeah... it seems like best practice to use `|| true` for yourself, at least. I'll leave it.